client/src/pages/signup.js

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import { Button, TextField, Paper, FormControl } from '@material-ui/core';
+import { Link } from "react-router-dom";
+import laptopPhoto from '../images/blue-shirt-at-laptop.png'
+import './signup.css'
+
+export default function Signup() {
+
+    const [firstName, setFirstName] = useState("");
+    const [lastName, setLastName] = useState("");
+    const [email, setEmail] = useState("");
+    const [password, setPassword] = useState("");
+    const [confirmPassword, setConfirmPassword] = useState("")
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        if (password === confirmPassword) {
+            axios.post('/signup', {
+                firstName: firstName,
+                lastName: lastName,
+                email: email,
+                password: password
+            })
+                .then(window.location = "/")
```

Comment:
We are using `react-router-dom` for routing in this project. Try to stay consistent with our routing methods to avoid problems later. You can use a conditional and the [Redirect](https://reactrouter.com/web/api/Redirect) component together in the `return` to redirect when a certain condition is met, in this case when the user has successfully logged in